ImportError: No module named google.protobuf error only in Travis

<!-- For general purpose questions, use Stack Overflow http://stackoverflow.com/questions/tagged/hyperledger -->
## Description

<!-- Describe your issue or user story in detail -->

Observed google.protobuf error in travis builds while executing behave tests. This issue observed in almost all the Travis PR Builds after merge the PR #2213 (commit https://github.com/hyperledger/fabric/commit/a511daa6bde7bef7e658c4059fc7b467dca871a1) in fabric master.  

```
Binary available as build/bin/peer
Running behave tests
Exception ImportError: No module named google.protobuf
Traceback (most recent call last):
  File "/usr/local/bin/behave", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/dist-packages/behave/__main__.py", line 109, in main
    failed = runner.run()
  File "/usr/local/lib/python2.7/dist-packages/behave/runner.py", line 672, in run
    return self.run_with_paths()
  File "/usr/local/lib/python2.7/dist-packages/behave/runner.py", line 678, in run_with_paths
    self.load_step_definitions()
  File "/usr/local/lib/python2.7/dist-packages/behave/runner.py", line 658, in load_step_definitions
    exec_file(os.path.join(path, name), step_module_globals)
  File "/usr/local/lib/python2.7/dist-packages/behave/runner.py", line 304, in exec_file
    exec(code, globals, locals)
  File "steps/bdd_grpc_util.py", line 20, in <module>
    import devops_pb2
  File "/home/travis/gopath/src/github.com/hyperledger/fabric/bddtests/devops_pb2.py", line 6, in <module>
    from google.protobuf import descriptor as _descriptor
ImportError: No module named google.protobuf
make: *** [behave] Error 1
rm build/docker/bin/membersrvc build/docker/bin/peer
```

commit a511daa6bde7bef7e658c4059fc7b467dca871a1
Author: Jeronimo jeronimo.irazabal@gmail.com
Date:   Tue Jul 19 11:40:25 2016 -0300
## Describe How to Reproduce

<!-- If an issue, provide sufficient context and steps to reproduce the issue -->

Travis Builds - 
https://travis-ci.org/hyperledger/fabric/jobs/146260865
https://travis-ci.org/hyperledger/fabric/jobs/145844081

--- a/scripts/provision/host.sh
+++ b/scripts/provision/host.sh
@@ -16,6 +16,7 @@ apt-get install --yes libyaml-dev
 
 apt-get install --yes python-setuptools
 apt-get install --yes python-pip
+pip install --upgrade pip
 pip install behave
 pip install nose
 
@@ -25,7 +26,7 @@ pip install -I flask==0.10.1 python-dateutil==2.2 pytz==2014.3 pyyaml==3.10 couc
 # Python grpc package for behave tests
 # Required to update six for grpcio
 pip install --ignore-installed six
-pip install 'grpcio==0.13.1'
+pip install --upgrade 'grpcio==0.13.1'
 
 # install ruby and apiaryio
 #apt-get install --yes ruby ruby-dev gcc

--- a/scripts/provision/host.sh
+++ b/scripts/provision/host.sh
@@ -26,7 +26,7 @@ pip install -I flask==0.10.1 python-dateutil==2.2 pytz==2014.3 pyyaml==3.10 couc
 # Python grpc package for behave tests
 # Required to update six for grpcio
 pip install --ignore-installed six
-pip install 'grpcio==0.15'
+pip install --upgrade 'grpcio==0.13.1'
 
 # install ruby and apiaryio
 #apt-get install --yes ruby ruby-dev gcc

--- a/scripts/provision/host.sh
+++ b/scripts/provision/host.sh
@@ -26,7 +26,7 @@ pip install -I flask==0.10.1 python-dateutil==2.2 pytz==2014.3 pyyaml==3.10 couc
 # Python grpc package for behave tests
 # Required to update six for grpcio
 pip install --ignore-installed six
-pip install --upgrade 'grpcio==0.13.1'
+pip install --upgrade 'grpcio==0.15'
 
 # install ruby and apiaryio
 #apt-get install --yes ruby ruby-dev gcc

--- a/scripts/provision/host.sh
+++ b/scripts/provision/host.sh
@@ -26,7 +26,7 @@ pip install -I flask==0.10.1 python-dateutil==2.2 pytz==2014.3 pyyaml==3.10 couc
 # Python grpc package for behave tests
 # Required to update six for grpcio
 pip install --ignore-installed six
-pip install --upgrade 'grpcio==0.15'
+pip install 'grpcio==0.15'
 
 # install ruby and apiaryio
 #apt-get install --yes ruby ruby-dev gcc

